src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,71 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  filterSelection: string;
+  setFilterSelection: (selection: string) => void;
+  query: string;
+  setQuery: (query: string) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  filterSelection,
+  setFilterSelection,
+  query,
+  setQuery,
+}) => {
+  const handleFilterSelection
+    = (event: React.ChangeEvent<HTMLSelectElement>) => {
+      event.preventDefault();
+      setFilterSelection(event.target.value);
+    };
```

Comment:
I didn't do prevent default here. Please, explain me, what do you prevent?

Reply:
agree here it redundant